deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -0,0 +1,363 @@
+import requests
+import yaml
+
+from typing import List, Optional
+from .constants import WAZUH_CONF, WAZUH_ROOT
+from .executor import Executor, WazuhAPI
+from .generic import HostInformation, CheckFiles
+
+
+class WazuhAgent:
+
+    @staticmethod
+    def install_agent(inventory_path, agent_name, wazuh_version, wazuh_revision, live) -> None:
+
+        if live == True:
+            s3_url = 'packages'
+            release = wazuh_version[0:3]
+        else:
+            s3_url = 'packages-dev'
+            release = 'pre-release'
+
+        os_type = HostInformation.get_os_type(inventory_path)
+        commands = []
+        if 'linux' in os_type:
+            distribution = HostInformation.get_linux_distribution(inventory_path)
+            architecture = HostInformation.get_architecture(inventory_path)
+
+            if distribution == 'rpm' and 'x86_64' in architecture:
+                commands.extend([
+                    f"curl -o wazuh-agent-{wazuh_version}-1.x86_64.rpm https://{s3_url}.wazuh.com/{release}/yum/wazuh-agent-{wazuh_version}-1.x86_64.rpm && sudo WAZUH_MANAGER='MANAGER_IP' WAZUH_AGENT_NAME='{agent_name}' rpm -ihv wazuh-agent-{wazuh_version}-1.x86_64.rpm"
```

Comment:
Is MANAGER_IP replaced at some point? If not, what is the purpose of installing with variables that are not used?
Same question for all remaining iterations

Reply:
The execution is done in 2 stages since on the one hand the agent is installed and then in the registration test the connection is made with the manager by changing the IP